apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/utils.ts

```diff
@@ -21,6 +21,17 @@ export const redirectToWpcomPath = ( url: string ): void => {
 	redirectParentWindow( `${ origin }${ path }` );
 };
 
+/**
+ * Opens the checkout.
+ *
+ * If possible, the checkout is opened as a modal without a page redirect (for Focused Launch).
+ * Otherwise, as a fallback, the user is redirected to the /checkout page or the /home page
+ * (in this case, `siteSlug` and `isEcommerce` params are used to construct the redirect url)
+ *
+ * @param siteSlug The slug (id) of the current site.
+ * @param isEcommerce True if the eCommerce plan is going to be in the checkout.
+ * @param onSuccessCallback Called when the checkout opens as a modal and is completed successfully
+ */
 export const openCheckout = (
 	siteSlug: string = window?.currentSiteId?.toString() || '',
```

Comment:
Just verified. `window.currentSiteId` doesn't exist. `window._currentSiteId` does. Then we can also fix the typings.

Reply:
Cleaned up that variable in https://github.com/Automattic/wp-calypso/pull/50182/commits/4d9ec5fe6dffee874d9096f3a5c13c4021b61b2c

Reply:
> @yansern can you have another look?

Looks good! :)